tensorflow/compiler/mlir/hlo/BUILD

```diff
@@ -1693,6 +1694,24 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "buffer_reuse",
+    srcs = ["lib/Transforms/buffer_reuse.cc"],
+    hdrs = [
+        "include/mlir-hlo/Analysis/userange_analysis.h",
+        "include/mlir-hlo/Transforms/PassDetail.h",
+        "include/mlir-hlo/Transforms/passes.h",
+    ],
+    deps = [
+        ":hlo",
+        ":transforms_pass_inc_gen",
+        "@llvm-project//mlir:MemRefDialect",
```

Comment:
These are ordered incorrect. See [the builder log](https://source.cloud.google.com/results/invocations/023880b0-268d-42dd-ae99-d6b088e90f2a/targets/%2F%2Ftensorflow%2Ftools%2Fci_build:gen_ci_sanity_out/log).